rotkehlchen/chain/ethereum/modules/curve/decoder.py

```diff
@@ -286,15 +322,29 @@ def _maybe_enrich_curve_transfers(
             event.counterparty = CPT_CURVE
             event.notes = f'Receive {event.balance.amount} {crypto_asset.symbol} from the curve pool {CURVE_Y_DEPOSIT}'  # noqa: E501
             return True
+        if (
+            event.event_type == HistoryEventType.RECEIVE and
+            event.event_subtype == HistoryEventSubType.NONE and
+            source_address in self.curve_gauges
+        ):
+            crypto_asset = event.asset.resolve_to_crypto_asset()
+            event.event_subtype = HistoryEventSubType.REWARD
+            event.notes = f'Receive {event.balance.amount} {crypto_asset.symbol} rewards from curve'  # noqa: E501
```

Comment:
Add the curve gauge or name here

```suggestion
            event.notes = f'Receive {event.balance.amount} {crypto_asset.symbol} rewards from curve'  # noqa: E501
```